scripts/branch_rename_migration.py

```
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
```

Pylint (4.0.2)

[refactor] Useless option value for '--disable', 'C0330' was removed from pylint, see https://github.com/pylint-dev/pylint/pull/3571.

(R0022)